.github/workflows/docker.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: '>=1.18'
+          go-version: '1.18'
```

Comment:
for the official builds, we'd like to stick with the latest stable version of Go.

Reply:
Do you mean to buildtest use 1.18，and release build by the last stable?

Reply:
yup, buildtest makes sense to be 1.18, since it's enforcing the 1.18 minimum.